indigo_api/serializers.py

```diff
@@ -550,9 +563,9 @@ class Meta:
             # readonly, url is part of the rest framework
             'id', 'url',
             'title', 'publication_name', 'publication_number', 'publication_date', 'publication_document',
-            'commenced', 'commencement_date', 'assent_date', 'stub',
+            'commenced', 'assent_date', 'stub',
```

Comment:
I'm looking at how useful it is for us in our code to have easy access to the commencement date (and possibly other details) of the main commencement. I wonder if we should retain these fields in the API and populate them from the main commencement object. It would make the API backwards compatible, and make working with the commencement common case (commenced, single work), much simpler. Otherwise, an API user must iterate over the commencement objects, find the main one, and work with that.